contracts/deploy/06_deploy_TokenBridge.ts

```diff
@@ -88,7 +88,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const tokenBridge = await upgrades.deployProxy(TokenBridgeFactory, [
     {
-      defaultAdmin: tokenBridgeSecurityCouncil,
+      defaultAdmin: securityCouncilAddress,
```

Comment:
Shouldn't we have the same change in `contracts/local-deployments-artifacts/deployBridgedTokenAndTokenBridge.ts`?

Reply:
we do - it is on the global setup

Reply:
good point - fixed